src/Microsoft.DotNet.ProjectModel/Files/IncludeContext.cs

```diff
@@ -38,7 +38,15 @@ public class IncludeContext
             SourceBasePath = sourceBasePath;
             Option = option;
             var token = rawObject.Value<JToken>(option);
-            if (token.Type != JTokenType.Object)
+
+            if (token == null)
+            {
+                IncludePatterns = new List<string>();
+                ExcludePatterns = new List<string>();
+                IncludeFiles = new List<string>();
+                ExcludeFiles = new List<string>();
```

Comment:
IIRC I needed to make this change because the default include/exclude files (`**/*.cs`) were not getting populated when this token was null. My code handles the null case. 

I can't seem to retrace that bit of code, but I remember it taking a while to find in the first place. I'll try to trace it down more.